lib/autoproj/vcs_definition.rb

```diff
@@ -307,7 +307,7 @@ def self.call_source_handler(vcs, url, options)
         if !handler
             raise ArgumentError, "there is no source handler for #{vcs}"
         else
-            return handler.call(url, options)
+            return handler.call(url, **options)
```

Comment:
explicit return statement?

Reply:
I didn't go through the rubocop errors ... just the 2.7 warnings.